src/lib.rs

```diff
@@ -282,6 +284,47 @@ pub type DMat3 = DMat3x3;
 /// Matrix of f64s with 4 columns and 4 rows. Alignment 32, size 128.
 pub type DMat4 = DMat4x4;
 
+/// Pads an element to be in an array in a shader.
+///
+/// All elements in arrays need to be aligned to 16s. This automatically aligns your types to 16s.
+///
+/// This glsl:
+///
+/// ```glsl
+/// struct FloatArray {
+///     float array[16];
+/// };
+/// ```
+///
+/// turns into:
+///
+/// ```rust
+/// #[repr(C)]
+/// struct FloatArray {
+///     array: [shader_types::ArrayMember<f32>; 16]
+/// }
+/// ```
+#[repr(C, align(16))]
+#[derive(Debug, Copy, Clone, Default, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct ArrayMember<T>(pub T);
+
+#[cfg(feature = "bytemuck")]
+unsafe impl<T: bytemuck::Zeroable> bytemuck::Zeroable for ArrayMember<T> {}
+#[cfg(feature = "bytemuck")]
+unsafe impl<T: bytemuck::Pod> bytemuck::Pod for ArrayMember<T> {}
+
+/// Pads a structure for use with dynamic offsets in wgpu.
+///
+/// All dynamic offsets need to be aligned to 256s. This automatically aligns your types to 256s.
```

Comment:
```suggestion
/// All dynamic offsets need to be aligned to 256 bytes. This automatically aligns your types to 256 bytes.
```
Same suggestion as above.